Vocabularies/DomHonorific.js

```diff
@@ -1,7 +1,5 @@
 function domHonorificVocabulary() {
-    //TODO: Choose title at the beginning (random/selection)?
-    const answers = ["Mistress", "Princess", "Goddess", "Maitrese" ,"Miss"];
+    const answers = ["Mistress", "Princess", "Mistress", "Goddess", "Maitrese" ,"Miss"];

```

Comment:
Why remove the todo and why randomize it? Makes no sense that your domme has a different honorific every time

Reply:
the intent was to randomize the vocabulary usage (like all the other vocabularies....)  existing code was hardcoded to a single choice??? how is that better ( and you already have %domhonorrific% for a chosen singular honorific.)